.teamcity/Gradle_Check/model/CIBuildModel.kt

```diff
@@ -284,8 +284,9 @@ enum class JvmVendor {
 }
 
 enum class PerformanceTestType(val taskId: String, val timeout : Int, val defaultBaselines: String = "", val extraParameters : String = "") {
-    test("PerformanceTest", 420, "defaults"),
+    test("PerformanceTest", 420, "nightly"),
     experiment("PerformanceExperiment", 420, "defaults"),
+    flakinessDetection("FlakinessDetection", 420, "nightly"),
```

Comment:
So we are running flakiness detection against the latest nightly now, which automatically turns of caching?

Would it be much harder to use the current commit as a baseline?

Reply:
Now I introduced a new baseline `flakiness-detection-commit` and build cache is disabled upon this baseline. It will be resolved to current commit.